src/pages/EditMonitor.vue

```diff
@@ -385,6 +388,23 @@
                                 </div>
                             </template>
 
+                            <template v-if="monitor.type === 'zabbix-trigger'">
+                                <div class="my-3">
+                                    <label for="zabbix_instance_url" class="form-label">{{ $t("zabbixInstanceUrl") }}</label>
+                                    <input id="zabbix_instance_url" v-model="monitor.zabbixInstanceUrl" type="text" class="form-control" required />
```

Comment:
I think here a placeholder what the url we expect is might be helpfull.

```suggestion
                                    <input id="zabbix_instance_url" v-model="monitor.zabbixInstanceUrl" type="text" class="form-control" placeholder="https://example.com/zabbix/api_jsonrpc.php" required />
```